src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -18,13 +23,20 @@ export const useTradeRoutes = (): {
   const buyAsset = getValues('buyAsset')
   const sellAsset = getValues('sellAsset')
   const assets = useSelector(selectAssets)
-  const feeAsset = assets['eip155:1/slip44:60']
+  const feeAsset = assets[ETHEREUM_CAIP19]
 
   const setDefaultAssets = useCallback(async () => {
     try {
       const [sellAssetId, buyAssetId] = getDefaultPair()
       const sellAsset = assets[sellAssetId]
-      const buyAsset = assets[buyAssetId]
+      // TODO: Actually we only support ERC20 pairs, but for example we will need to support cosmos pairs soon
+      const buyAsset =
+        defaultBuyAssetId &&
+        assets[defaultBuyAssetId]?.chain === ChainTypes.Ethereum &&
```

Comment:
We don't need to explicitly check for the asset chain if we check for the CAIP19 since it's already include:
https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md#syntax

The "chain_id" part is the CAIP2, which is effectively what you're checking here